ix-dev/test/glances/questions.yaml

```diff
@@ -0,0 +1,139 @@
+          label: Additional Environment Variables
+          description: Configure additional environment variables for Glances.
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: env
+                label: Environment Variable
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: name
+                      label: Name
+                      schema:
+                        type: string
+                        required: true
+                    - variable: value
+                      label: Value
+                      schema:
+                        type: string
+                        required: true
+
+  # - variable: network
+  #   label: ""
+  #   group: Network Configuration
+  #   schema:
+  #     type: dict
+  #     attrs:
+  #       - variable: web_port
+  #         label: Webserver Port
+  #         description: The port for the Glances webserver.
+  #         schema:
+  #           type: int
+  #           default: 30040
+  #           required: false
+  #           $ref:
+  #             - definitions/port
+
+  - variable: labels
+    label: ""
+    group: Labels Configuration
+    schema:
+      type: list
+      default: []
+      items:
+        - variable: label
+          label: Label
+          schema:
+            type: dict
+            attrs:
+              - variable: key
+                label: Key
+                schema:
+                  type: string
+                  required: true
+              - variable: value
+                label: Value
+                schema:
+                  type: string
+                  required: true
+              - variable: containers
+                label: Containers
+                description: Containers where the label should be applied
+                schema:
+                  type: list
+                  items:
+                    - variable: container
+                      label: Container
+                      schema:
+                        type: string
+                        required: true
+                        enum:
+                          - value: glances
+                            description: glances
+
+  - variable: resources
+    label: ""
+    group: Resources Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: limits
+          label: Limits
+          schema:
+            type: dict
+            attrs:
+              - variable: cpus
+                label: CPUs
+                description: CPUs limit for Wordpress.
+                schema:
+                  type: int
+                  default: 1
+                  required: true
+              - variable: memory
+                label: Memory (in MB)
+                description: Memory limit for Wordpress.
+                schema:
+                  type: int
+                  default: 256
+                  required: true
```

Comment:
Please keep default values same as the rest of the apps.
It makes it impossible to mass edit when needed.
if there other places that defaults are changed please adjust.